### DISCUSSION TITLE
Pretrained model for entity linking

### DISCUSSION THREAD

**Discussion by dzieciou** [explosion/spaCy#5156] [Help: Other Questions]
## Feature description

Is there a pretrained model for Entity Linking using Wikipedia/WikiData as DB?

Spacy recently started to support Entity Linking [1],[2],[3] but they do not provide pretrained model for that and some people report training takes at least a week [4]. 

## Could the feature be a [custom component](https://spacy.io/usage/processing-pipelines#custom-components) or [spaCy plugin](https://spacy.io/universe)?
If so, we will tag it as [`project idea`](https://github.com/explosion/spaCy/labels/project%20idea) so other users can take it on.

I guess it should a separate model file.

[1]: https://spacy.io/usage/linguistic-features#entity-linking
[2]: https://github.com/explosion/spaCy/tree/master/bin/wiki_entity_linking
[3]: https://github.com/explosion/spaCy/issues/4511
[4]: https://stackoverflow.com/questions/58541373/training-times-for-spacy-entity-linking-model

**Comment by svlandeg** [comment]
No, that's currently not readily available, the feature is still in beta development. You can use the example scripts provided to train your own, on Wikipedia/Wikidata or any other resource. Obviously it depends on which data you train it on, and how much of it, how long the training will take. I don't think it is not necessary for most use-cases to train it on the whole of English Wikipedia, but that's your choice. Assembling the KB and training on 165.000 articles, took me a few days total.

See also the discussion here: https://github.com/explosion/spaCy/issues/4511
